sbc_compassion/data/child_layouts.xml

```diff
@@ -130,5 +167,25 @@
             <field name="code">6S11</field>
             <field name="additional_page_box_id" ref="l6_box_1"/>
         </record>
+
+        <record id="b2s_l7" model="correspondence.b2s.layout">
+            <field name="name">L7</field>
```

Comment:
I'm just afraid that we will forget what these new templates are for. Can you just rename them to be more explicit?
```suggestion
            <field name="name">L7 Intro 9-11</field>
```